arcade/gl/glsl.py

```diff
@@ -1,7 +1,9 @@
-from typing import Dict, Iterable, List, Optional
+from typing import TYPE_CHECKING, Dict, Iterable, List, Optional
 import re
 
 from pyglet import gl
+if TYPE_CHECKING:
+    from .context import Context as ArcadeGlContext
```

Comment:
I would have expected this code to upset the linter, as imports aren't supposed to come after other code? I'm ok since it appears to work.

Reply:
Perhaps `if TYPE_CHECKING:` is special-cased, since the Python ecosystem recommends this pattern.